### PR TITLE
Fix worldCopyJump with Keyboard

### LINKED ISSUE #8560: Pan with keyboard and dragging with mouse gives different result when passing dateline
### Checklist

- [X] I've looked at the [documentation](https://leafletjs.com/reference.html) to make sure the behavior isn't documented and expected.
- [X] I'm sure this is an issue with Leaflet, not with my app or other dependencies (Angular, Cordova, React, etc.).
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this hasn't been reported yet.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Steps to reproduce

Hi
_If I add map1.on('moveend', function(event){  console.log('moveend lng=',this.getCenter().lng); });_
and drag the map passed the dateline (lng=180) I get (almost) the correct mapCenter, but if I pan the map using the keyboard I get lng > 180 if I pan west to east. See attached video

### Expected behavior

I would expect the resulting lng to be the same, and also < 180

### Current behavior

https://user-images.githubusercontent.com/11357972/195583584-02b474d7-eaf0-4572-bd69-299740b1fc87.mov

### Minimal example reproducing the issue

_No response_

### Environment

- Leaflet version: 1.9.2
- Browser (with version): Chrome
- OS/Platform (with version): Windows

### PR DESCRIPTION
**Pull request by Falke-Design**
Fixes #8560

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@Falke-Design is this something we would like to back-port as well?

**Comment by Falke-Design** [comment]
> @Falke-Design is this something we would like to back-port as well?

Yes I think we can back-port this too

File: src/map/handler/Map.Keyboard.js [hunk 1]
```diff
@@ -151,10 +151,15 @@ export var Keyboard = Handler.extend({
...
 
-				map.panBy(offset);
-
 				if (map.options.maxBounds) {
-					map.panInsideBounds(map.options.maxBounds);
+					offset = map._limitOffset(toPoint(offset), map.options.maxBounds);
+				}
+
+				if (map.options.worldCopyJump) {
+					var newLatLng = map.wrapLatLng(map.unproject(map.project(map.getCenter()).add(offset)));
```
Comment (IvanSanchez) [inline src/map/handler/Map.Keyboard.js:159]: This looks like something that could be moved to the `Map`'s `panBy()` method.

Comment (Falke-Design) [reply src/map/handler/Map.Keyboard.js:159]: I also thought about that but I do not completly understand how the original wrapping in Map.Drag works.

Comment (Falke-Design) [reply src/map/handler/Map.Keyboard.js:159]: I digged into it, it is not so easy. I created a new issue / task for this. #8625

**Pull request merged**